src/geobr.jl

```diff
@@ -0,0 +1,645 @@
+
+Get semiarid data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2017).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Semiarid data for the specified year.
+"""
+semiarid(; year=2017, kwargs...) = get("semiarid", year, nothing, nothing, kwargs...)
+
+"""
+    schools(; year=2020, kwargs...)
+
+Get schools data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2020).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Schools data for the specified year.
+"""
+schools(; year=2020, kwargs...) = get("schools", year, nothing, nothing, kwargs...)
+
+"""
+    comparableareas(; startyear=1970, endyear=2010, kwargs...)
+
+Get comparable areas data for a given range of years.
+
+Arguments:
+- `startyear`: Start year of the data (default: 1970).
+- `endyear`: End year of the data (default: 2010).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Comparable areas data for the specified range of years.
+"""
+function comparableareas(; startyear=1970, endyear=2010, kwargs...)
+    years_available = [1872, 1900, 1911, 1920, 1933, 1940, 1950, 1960, 1970, 1980, 1991, 2000, 2010]
+
+    if startyear ∉ years_available || endyear ∉ years_available
+        throw(ArgumentError("Invalid `startyear` or `endyear`. It must be one of the following: $years_available"))
+    end
+
+    metadata = metadatarows("amc", startyear, nothing, nothing; returnall=true)
+    metadata = metadata |> Filter(row -> contains(row.download_path, "$(startyear)_$(endyear)")) |> Tables.rows
+
+    if isempty(metadata)
+        throw(ErrorException("No data found for the specified years"))
+    end
+
+    first_row = first(metadata)
+    get("amc", first_row.year, nothing, nothing; kwargs...)
+end
+
+"""
+    urbanconcentrations(; year=2015, kwargs...)
+
+Get urban concentrations data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2015).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Urban concentrations data for the specified year.
+"""
+urbanconcentrations(; year=2015, kwargs...) = get("urban_concentrations", year, nothing, nothing, kwargs...)
+
+"""
+    poparrangements(; year=2015, kwargs...)
+
+Get population arrangements data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2015).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Population arrangements data for the specified year.
+"""
+poparrangements(; year=2015, kwargs...) = get("pop_arrengements", year, nothing, nothing, kwargs...)
+
+"""
+    healthregion(; year=2013, kwargs...)
+
+Get health region data for a given year.
+
+Arguments:
+- `year`: Year of the data (default: 2013).
+- `kwargs`: Additional keyword arguments.
+
+Returns:
+- Health region data for the specified year.
+"""
+healthregion(; year=2013, kwargs...) = get("health_region", year, nothing, nothing, kwargs...)
+
+function filepath(ID, url)
+    filename = split(url, "/") |> last
+    ID * "/" * filename
+end
```

Comment:
```suggestion
filepath(ID, url) = joinpath(ID, basename(url))
```

please double check if `basename` works with general urls